README.md

```diff
@@ -1,82 +1,104 @@
 # Red Hat iPaaS OpenShift Templates
 
-This repository contains a simple way to get the Red Hat iPaaS deployed, using OpenShift templates,
-on a running cluster.
+This repository contains a simple way to get the Red Hat iPaaS deployed, using OpenShift templates, on a running cluster.
 
-Run the following commands:
+There are several flavors of templates you can use (see [below]()), but in general in order to apply the templates you can directly refer to the given files via a GitHub URL:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/redhat-ipaas/openshift-templates/master/redhat-ipaas.yml
-$ oc new-app redhat-ipaas -p ROUTE_HOSTNAME=<external hostname>
+$ oc create -f https://raw.githubusercontent.com/redhat-ipaas/openshift-templates/master/ipaas.yml
 ```
 
-Replace _&lt;external hostname&gt;_ with a value that will resolve to the address of the OpenShift router.
-
-You have to chose an address or _&lt;external hostname&gt;_ which is routable on your system (and also resolvable from inside your cluster). For a development setup you can use an external DNS resolving service like xip.io or nip.io:
-Assuming that your OpenShift cluster is reachable under the IP address _ip_ then use `ipaas.`_ip_`.nip.io`.) (e.g. `ipass.127.0.0.1.nip.io` if your cluster is listening on localhost). With minishift you can retrieve the IP of the cluster with `minishift ip`.
-
-Once all pods are started up, you should be able to access the iPaaS at `https://`_&lt;external hostname&gt;_`/`.
-
 ## Template flavours
 
 Currently there are multiple different flavours of the templates, with the following characteristics:
 
-* `dev`: Are using pure docker images, instead of image streams.
-* `ephemeral:` Are not using persistence. Mostly needed for testing as a workaround to the [pods with pvc sporadically timeout](https://bugzilla.redhat.com/show_bug.cgi?id=1435424) issue .
-* `single-tenant`: As the name implies.
+| Template | Descripton |
+| ipaas.yml | Full production when setting up on a cluster with full access rights. Uses image streams under the hoods. |
+| ipaas-dev.yml | Same as above, but with direct references to Docker images so that they locally created images (e.g. agains a Minishift Docker daemon) can be used directly |
```

Comment:
Hi @rhuss - sorry for nitpicking but it looks like there is some typo which breaks table formatting.
![screenshot_20170405_093728](https://cloud.githubusercontent.com/assets/8707241/24694802/a4cfbfae-19e3-11e7-86b3-2ad01e95ccf3.png)